src/OrbitDB.js

```diff
@@ -260,6 +274,23 @@ class OrbitDB {
       await this._pubsub.unsubscribe(address)
     }
 
+    // Cache cleanup
+    // First remove the handler by db address
+    // Then close the cache if there are no more handlers
+    for(let directory of Object.keys(this.caches)) {
+      const cache = this.caches[directory] || this.cache
+      const store = this.stores[address]
+
+      if (store && store.options.directory == directory) {
+        // Closing a custom cache?
+        if (cache.handlers.has(address)) cache.handlers.delete(address)
```

Comment:
Can this be handled via some interface in `orbit-db-cache`? The `handlers` property shouldn't be directly manipulated from orbitdb imo

Reply:
Perhaps we could call sth like `this.cache.rmHandler(address)`, the check can be done in `rmHandler` and the cache can close itself if its handlers size is 0. What do you think?